app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue

```diff
@@ -203,6 +205,16 @@ export default {
     facebookUnauthorized() {
       return this.isAFacebookInbox && this.inbox.reauthorization_required;
     },
+    googleUnauthorized() {
+      const isLegacyInbox = ['imap.gmail.com', 'imap.google.com'].includes(
```

Comment:
Based on the code, I believe we're going to mark the email inboxes with a "reauthorization_required" flag. Is that correct? Will this apply to all Gmail accounts, or is there a way to identify only the older accounts that need reauthorization?

Reply:
>  I believe we're going to mark the email inboxes with a "reauthorization_required" flag
Yes

> or is there a way to identify only the older accounts that need reauthorization?
It will only apply to legacy ones given we mark them specifically. We don't really need to identify the older accounts ourselves.